paasta_tools/api/api.py

```diff
@@ -53,6 +53,11 @@ def parse_paasta_api_args():
         dest="soa_dir",
         help="define a different soa config directory",
     )
+    parser.add_argument(
```

Comment:
Why do we need to be able to run a PaaSTA API server for a different cluster?

Reply:
This makes it easier to test. kwa suggested to add this.